pkg/destroy/gcp/dns.go

```diff
@@ -87,6 +87,10 @@ func (o *ClusterUninstaller) deleteDNSZoneRecordSets(zoneName string, zoneDomain
 		o.resetRequestID("recordsets", zoneName)
 		return errors.Wrapf(err, "failed to delete DNS zone %s recordsets", zoneName)
 	}
+	if change == nil {
```

Comment:
As the result is the same as the previous conditional you could fold this into the previous conditional statement:
```go
if (err != nil && !isNoOp(err)) || change == nil {
```

Reply:
Thanks for the recommendation. However, not the same... no `Wrapf(err`